### DISCUSSION TITLE
Account change password

### DISCUSSION THREAD

**Discussion by antoinechamot** [keycloak/keycloak#15599] [Q&A]
Hi,

I am developping a vue application protected by keycloak using keycloak-js.
I have a requirement that each user have an account page in my app to change their password
![image](https://user-images.githubusercontent.com/31971238/203074956-39faa805-31cd-4308-80c1-265615a772e8.png)

How can i do that ? I don't find any endpoint with old_password,new_password,confirmation in the rest api documentation

**Comment by antoinechamot** [comment]
Nobody did that on his project ?

**Reply by sschu** [reply]
Keycloak implements that using a required action and performs the password itself. I am not aware of an extra endpoint that would allow this.
